docs/topics/braintree.rst

```diff
@@ -227,6 +227,16 @@ solitude.
 
     :status 201: payment method created.
 
+Change payment method on a subscription:
+
+.. http:post:: /braintree/subscription/change/
```

Comment:
what data do you have in the payments-service, the URL to the solitude subscription?